modules/ROOT/pages/_partials/data-sync/js-client-offline.adoc

```diff
@@ -289,15 +303,14 @@ const query = client.watchQuery<AllTasks>({
 query.subscribeToMore(subscriptionOptions);
 ----
 
-The cache will now be kept up to date with automatic data deduplication being performed.
+The cache is kept up to date while automatically performing data deduplication.
 
-===== Multiple Subscriptions
+=== Using cache update helpers for multiple subscriptions
 
-{sync-client} also provides the ability to automatically call `subscribeToMore` on your `ObservableQuery`. This can be useful
-in a situation where you may have multiple subscriptions which can affect one single query. For example, if you have a
-`TaskAdded`, `TaskDeleted` and a `TaskUpdated` subscription you would need three separate `subscribeToMore` function calls. This can become
-tedious as your number of subscriptions grow. To combat this, we can use the `subscribeToMoreHelper` function from {sync-client} to automatically handle this for us by passing
-it an array of subscriptions and their corresponding queries which need to be updated.
+The @aerogear/voyager-client module provides the ability to automatically call `subscribeToMore` on your `ObservableQuery`. 
+This can be useful in a situation where you may have multiple subscriptions which can affect one single query. 
+For example, if you have a `TaskAdded`, `TaskDeleted` and a `TaskUpdated` subscription you  need three separate `subscribeToMore` function calls. 
+To avoid this, use the `subscribeToMoreHelper` function from the @aerogear/voyager-client module to automatically handle this by passing  an array of subscriptions and their corresponding queries:
```

Comment:
```suggestion
To avoid this, use the `subscribeToMoreHelper` function from the @aerogear/voyager-client module to automatically handle this by passing an array of subscriptions and their corresponding queries:
```